src/sonic-config-engine/sonic-cfggen

```diff
@@ -301,7 +301,7 @@ def main():
             }}}
         deep_update(data, hardware_data)
         if args.port_config is None:
-            args.port_config = device_info.get_path_to_port_config_file()
+            args.port_config = device_info.get_path_to_port_config_file(hwsku)
```

Comment:
You defined the order of the parameters as `get_path_to_port_config_file(asic=None, hwsku=None)`, this `hwsku` parameter will be passed in as `asic`

Reply:
fixed